### PR TITLE
Added separator parameter to array function

### PR DESCRIPTION
**Pull request by seal-mt**
Hi @ctavan, here is the separate PR for extending the array function by a `separator` argument as requested.

Thanks
Manfred

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -95,20 +95,21 @@ Object.keys(convert).forEach(function(type) {
...
-  return value.split(/\s*,\s*/).map(convert[type]);
+  return value.split(separator).map(convert[type]);
 };
 
 getenv.multi = function multi(spec) {
   const result = {};
   for (let key in spec) {
     const value = spec[key];
-    if (util.isArray(value)) {
+    if (Array.isArray(value)) {
```
Comment (ctavan) [inline index.js:112]:
```suggestion
    if (util.isArray(value)) {
```

**Review by ctavan — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (ctavan) [inline index.js:112]: I think `util` is no longer used now. If you do this change here, then please remove the unused dependency.

**Review by ctavan — CHANGES_REQUESTED** [review]
Apologies for the long delay.

Looks very good in general. Just one think: Seems like you removed the last usage of `util`, so the `require` statement should also be removed.

File: index.js (hunk 1 above)
Comment (seal-mt) [reply index.js:112]: Yes you are right, i overlooked the require statement. Update is coming...

**Review by ctavan — APPROVED** [review]
LGTM

**Pull request merged**